src/index.d.ts

```diff
@@ -81,8 +81,10 @@ interface Inputs<T, Name = keyof T> {
   checkbox(name: Name, ownValue?: OwnValue): CheckboxProps<T>;
   checkbox(options: InputOptions<T, Name, Maybe<OwnValue>>): CheckboxProps<T>;
 
-  raw<Y>(name: Name, value?: Y): RawInputProps<Y>;
-  raw<Y>(options: RawInputOptions<T, Y, Name>): RawInputProps<Y>;
+  raw<RawValue>(name: Name): RawInputProps<T, Name, RawValue>;
```

Comment:
The raw value was inferred from the 2nd `value` argument which is incorrect. `value` is the input's "own value" which is very unlikely to be set with `raw`. When it's not set it'll leave the raw value untyped.

```diff
- raw<Y>(name: Name, value?: Y): RawInputProps<Y>;
+ raw<RawValue>(name: Name): RawInputProps<T, Name, RawValue>;
```

`value`/`ownValue` is usually used with inputs that have the same name but can support multiple values. It's only needed for checkboxes and radio buttons.

I'm removing `value` (the input's "ownValue") for now because it's intended for checkboxes and radio inputs where the `value` prop isn't the actual input values (in this case, for example, the actual value is `checked`).